### PR TITLE
Unify -h for fisher_list

### PR DESCRIPTION
**Pull request by pickfire**

### CODE REVIEW

**Comment by pickfire** [comment]
Just forced pushed to this repository, more help updated.

**Comment by pickfire** [comment]
Ops, I am not sure how to add a new request, but I have add a method to speed up `__fisher_cache_list` for not using `sed`. Hope you can `cherry-pick` that for the moment.

File: functions/__fisher_cache_list.fish [hunk 1]
```diff
@@ -1,3 +1,3 @@
 function __fisher_cache_list
-    find -L $fisher_cache/* -maxdepth 0 -type d ^ /dev/null | sed 's|.*/||'
+    find -L $fisher_cache/* -printf '%f\n' -maxdepth 0 -type d ^ /dev/null
```
Comment (bucaran) [inline functions/__fisher_cache_list.fish]: I love this, but it does not work on OS X.

**Comment by pickfire** [comment]
Why close this? Is it merged?

**Comment by bucaran** [comment]
I think so. This closed when I merged your previous commit and pushed 1.0 if I am not mistaken.

Would you mind checking everything is displaying correctly?

**Comment by pickfire** [comment]
No, both the commits that I had showed here isn't merged, including the one that makes `__fish_cache_list` faster and the `-h` unification.

**Comment by bucaran** [comment]
Oh, that one I can't merge and I already explained why, you need check the comments. :wink: 

screen shot 2016-03-03 at 7 31 23 pm

File: functions/__fisher_cache_list.fish (hunk 1 above)
Comment (pickfire) [inline functions/__fisher_cache_list.fish]: Sorry, didn't saw this that day. Why it wouldn't work?

I guess you need to cherry-pick the `-h` unification commit out then.

Comment (bucaran) [inline functions/__fisher_cache_list.fish]: screen shot 2016-03-03 at 7 37 11 pm

**Pull request closed without merging**